main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
```

Comment:
Templates don't need to be in a single line...  You can definitely pass in an entire file content.  So I was just thinking that the template would be a separate file that just gets loaded with a `require` call and then fed right into the `_.template` with the json structure.